### PR TITLE
fix: add `{cursor}` to options that insert an `=`

### PR DESCRIPTION
**Pull request by SeparateRecords**
Related: https://github.com/withfig/fig/issues/1031

This was a quick regex find-replace, please let me know if I missed anything.

### CODE REVIEW

File: src/wp.ts [hunk 1]
```diff
@@ -69,8 +69,8 @@ const global_parameter_skip_plugins1: Fig.Option = {
 
 const global_parameter_skip_plugins2: Fig.Option = {
   name: "--skip-plugins",
-  insertValue: "--skip-plugins=",
-  displayName: "--skip-plugins=",
+  insertValue: "--skip-plugins={cursor}",
+  displayName: "--skip-plugins={cursor}",
```
Comment (mschrage) [inline src/wp.ts:73]: `{cursor}` shouldn't be in display name.

File: src/wp.ts [hunk 2]
```diff
@@ -85,8 +85,8 @@ const global_parameter_skip_themes1: Fig.Option = {
 
 const global_parameter_skip_themes2: Fig.Option = {
   name: "--skip-themes",
-  insertValue: "--skip-themes=",
-  displayName: "--skip-themes=",
+  insertValue: "--skip-themes={cursor}",
+  displayName: "--skip-themes={cursor}",
```
Comment (mschrage) [inline src/wp.ts:89]: `{cursor}` shouldn't be in display name.

**Review by mschrage — COMMENTED** [review]
Do we want to specify the `insertValue` like this? Or should we just set `requiresEquals: true`

**Comment by SeparateRecords** [comment]
`requiresEquals` should be set on these for sure. Will do that. It currently doesn't insert the equals or put the cursor in the right spot though, so this still has to be done

**Comment by SeparateRecords** [comment]
`requiresEquals: true` is now on options that insert `...={cursor}`

**Comment by mschrage** [comment]
Should `requiresEquals` automatically set the insertion text to `name={cursor}`?

**Comment by SeparateRecords** [comment]
I think so! There's an [old convo in the fig discord about exactly this actually](https://discord.com/channels/837809111248535583/839904572993568820/908897442878078977)

**Comment by fedeci** [comment]
@SeparateRecords do we have an issue in GH to track that?

**Review by fedeci — APPROVED** [review]

**Pull request merged**